src/services/userService.js

```diff
@@ -17,6 +21,50 @@ export default function UserService() {
     return correctness
   }
 
+  const getSexualPosibleMatches = (ref, gender, uid, search) => {
+    return ref.orderByChild(`interests/${gender}`).equalTo(true).once('value')
+      .then(users => {
+        const usersArray = []
+        users.forEach(queryUser => {
+          if (queryUser.key !== uid && search.includes(queryUser.val().gender)) {
+            const user = queryUser.val()
```

Comment:
Acá tampoco sé si es `const`, pero again :man_shrugging:

Reply:
es que es const para la referencia

Reply:
Good to know :+1: